src/components/Logo/Logo.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Image from 'next/image';
+import logo from '../../../public/img/logo.png';
+
+type logoType = {
+    logoSize: number;
+}
+
+export default function Logo({logoSize}:logoType ) {
+
+    return (
+        <div className='animate-spin'>
+            <Image src={logo} alt="logo" width ={logoSize} />
+        </div>
+    );
+}
```

Comment:
The image gets squashed when resizing the page, we should use a fixed layout to prevent this. The animation does not respect users who prefer reduced motion, an accessible class name would be `motion-safe:animate-spin`. I also do not think we need the extra `div` element here, we can directly apply the animation to the image.

I suggest adjusting this component to:

```tsx
export default function Logo({ size }: LogoType) {
  return (
    <Image
      src={logo}
      alt="React developer community Kenya logo"
      width={size}
      height={size}
      layout="fixed"
      className="motion-safe:animate-spin"
    />
  );
}
```